### PR TITLE
Ed: Headsign on stop time

### PR DESCRIPTION
**Pull request by azime**
- Headsign column added in StopTime
- Load Headsign in fusio_parser
- Persist Headsign
- Tests Added

Tickets: http://jira.canaltp.fr/browse/NAVP-233 and http://jira.canaltp.fr/browse/NAVP-234

### CODE REVIEW

File: source/ed/types.h [hunk 1]
```diff
@@ -339,6 +339,7 @@ struct StopTime {
     bool is_frequency;
     bool wheelchair_boarding;
     bool date_time_estimated;
+    std::string headsign = "";
```
Comment (xlqian) [inline source/ed/types.h]: `= ""` is not necessary

Comment (azime) [inline source/ed/types.h]: OK

**Comment by antoine-de** [comment]
Ok for me. 

@pbougue can you review it too ?

File: source/ed/ed_persistor.cpp [hunk 2]
```diff
@@ -912,7 +913,7 @@ void EdPersistor::insert_stop_times(const std::vector<types::StopTime*>& stop_ti
             LOG4CPLUS_INFO(logger, inserted_count<<"/"<< size_st <<" inserted stop times");
             this->lotus.prepare_bulk_insert("navitia.stop_time",
             {"id", "arrival_time", "departure_time", "local_traffic_zone", "odt", "pick_up_allowed", "drop_off_allowed",
```
Comment (pbougue) [inline source/ed/ed_persistor.cpp]: minor: line length

**Comment by pbougue** [comment]
Ok for me.

**Comment by azime** [comment]
refactoring

**Pull request merged**